test/keyboard-navigation.html

```diff
@@ -793,13 +793,18 @@
         });
 
         describe('with hidden columns', () => {
-          it('should skip over hidden column with right arrow', () => {
+          it('should skip over hidden column with right arrow', (done) => {
             grid._columnTree[0][1].hidden = true;
-            body.children[0].children[0].focus();
+            requestAnimationFrame(() => {
```

Comment:
nit: Rather than relying on raf and making the test async, use `flushGrid(grid)` here after `...hidden=true`. This should work for the other tests also.

Reply:
Done. Thanks.

Reply:
Should work also in `column.html` tests

Reply:
Removed. Thanks.